deployment/aks-periscope.yaml

```diff
@@ -108,30 +119,38 @@ data:
   AZURE_BLOB_ACCOUNT_NAME: # <accountName, base64 encoded>
   AZURE_BLOB_SAS_KEY: # <saskey, base64 encoded>
 ---
-apiVersion: v1 
-kind: ConfigMap 
+apiVersion: v1
+kind: ConfigMap
 metadata:
   name: containerlogs-config
   namespace: aks-periscope
 data:
   DIAGNOSTIC_CONTAINERLOGS_LIST: kube-system
 ---
-apiVersion: v1 
-kind: ConfigMap 
+apiVersion: v1
+kind: ConfigMap
 metadata:
   name: kubeobjects-config
   namespace: aks-periscope
 data:
   DIAGNOSTIC_KUBEOBJECTS_LIST: kube-system/pod kube-system/service kube-system/deployment
 ---
-apiVersion: v1 
-kind: ConfigMap 
+apiVersion: v1
+kind: ConfigMap
 metadata:
   name: nodelogs-config
   namespace: aks-periscope
 data:
   DIAGNOSTIC_NODELOGS_LIST: /var/log/azure/cluster-provision.log /var/log/cloud-init.log
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collectorflags-config
+  namespace: aks-periscope
+data:
+  FLAGS_LIST: # enable collector: OSM, SMI
```

Comment:
💡 `ARC` PR already introduces this flag: https://github.com/Azure/aks-periscope/pull/54/files#diff-cabcaac1a231e2d530ba374ca786e837a4c7f802f98c86299b93874aaf2e78eaR138

Reply:
This is good. Left a comment on PR #54 about making the flag name generic so that we can reuse it

Reply:
@shalier will update this flag and logic once PR 54 is merged